### ISSUE TITLE
Where can i find the readme file?

### ISSUE THREAD

**Issue by rajasaad7** [afriedman94/Honeypot-iOS#1]
I am unable to understand how this honeypot works but i am intrusted to work on it.
if you can guide me through this? where can i find readme file that make it easier to understand what we have to do to make it work.

**Comment by afriedman94** [comment]
This uses ThreatStream's MHN, documentation found here: 
https://github.com/threatstream/mhn/wiki

I used a Raspberry PI: 
https://github.com/threatstream/mhn/wiki/Deploying-Dionaea-on-a-Raspberry-Pi

API reference found here: 
https://github.com/threatstream/mhn/wiki/MHN-REST-APIs

**Comment by rajasaad7** [comment]
Yes, I need an iPhone honeypot to capture live attacks, so kind of intrusted on your repository.

**Comment by afriedman94** [comment]
I haven't created one. This is an old project and I would not recommend that you use it for production. Your best bet would be to follow the mhn wiki that I linked above and use my project's source as an example. The Wiki should provide all the info you need.
